src/main/kotlin/com/hj/leetcode/kotlin/problem2130/Solution.kt

```diff
@@ -0,0 +1,50 @@
+package com.hj.leetcode.kotlin.problem2130
+
+import com.hj.leetcode.kotlin.common.model.ListNode
+
+/**
+ * LeetCode page: [2130. Maximum Twin Sum of a Linked List](https://leetcode.com/problems/maximum-twin-sum-of-a-linked-list/);
+ */
+class Solution {
+    /* Complexity:
+     * Time O(N) and Space O(N) where N is the size of head;
+     */
+    fun pairSum(head: ListNode?): Int {
+        val values = head.values()
+        return maxTwinSum(values)
```

Comment:
Use comment to briefly describe the intent

Reply:
Resolved at https://github.com/hj-core/hj-leetcode-kotlin/pull/62/commits/c131a5e543671ee99b2ec8888dd90942ffbcabb2